bundle/testdata/empty.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "",
+  "version": "",
+  "description": "",
+  "invocationImages": null,
```

Comment:
Shouldn't these fields be removed 🤔 ?
I mean, should we add omitempty there too?

Reply:
invocationImages is a required property, so I don't think omitEmpty makes sense for this field.